python/llm/example/CPU/HF-Transformers-AutoModels/Model/mixtral/README.md

```diff
@@ -0,0 +1,45 @@
+# Mixtral
+In this directory, you will find examples on how you could apply BigDL-LLM INT4 optimizations on Mixtral models on [Intel CPUs](../README.md). For illustration purposes, we utilize the [mistralai/Mixtral-8x7B-Instruct-v0.1](https://huggingface.co/mistralai/Mixtral-8x7B-Instruct-v0.1) as a reference Mixtral model.
+
+## Requirements
+To run these examples with BigDL-LLM on Intel CPUs, we have some recommended requirements for your machine, please refer to [here](../README.md#recommended-requirements) for more information.
+
+**Important: Please make sure you have installed `transformers==4.36.0` to run the example.**
+
+## Example: Predict Tokens using `generate()` API
+In the example [generate.py](./generate.py), we show a basic use case for a Mixtral model to predict the next N tokens using `generate()` API, with BigDL-LLM INT4 optimizations on Intel CPUs.
+### 1. Install
+We suggest using conda to manage the Python environment. For more information about conda installation, please refer to [here](https://docs.conda.io/en/latest/miniconda.html#).
+
+After installing conda, create a Python environment for BigDL-LLM:
+```bash
+conda create -n llm python=3.9 # recommend to use Python 3.9
+conda activate llm
+
+# below command will install PyTorch CPU as default
+pip install torch==2.0.1 --index-url https://download.pytorch.org/whl/cpu
```

Comment:
Maybe we could directly use `pip install --pre --upgrade bigdl-llm[all]` to install.

Reply:
changed to `bigdl-llm[all]`